src/grid/grid.css

```diff
@@ -4,7 +4,8 @@
 @value large-screen-media, middle-screen-media, small-screen-media from "../global/global.css";
 
 .container-fluid,
-.container {
+.container
```

Comment:
a comma is missing here, so it's `.container .row` instead of `.container, .row`

Reply:
I'll fix that

Reply:
Ouch I'm sorry, thank you for your help!